app/models/katello/authorization/content_view.rb

```diff
@@ -57,6 +57,10 @@ def publishable?
         ::User.current.can?(:publish_content_views)
       end
 
+      def exportable?
```

Comment:
I don't think any one is calling this. Can ya confirm and remove if that's the case (i just `git grep exportable app` and couldnt find any  references.)